examples/evkbmimxrt1060/defender/main.c

```diff
@@ -51,7 +51,7 @@
 
 #include "ksdk_mbedtls.h"
 
-#include "nxlog_App.h"
+#include "nxLog_App.h"
```

Comment:
Why switch this include file to be case sensitive, and the other to not be case sensitive? (are we adopting a policy for platform portability, e.g. some filesystems don't support case sensitivity and some do.)

Reply:
I think this is for code consistency. Some of code use "nxlog_App.h" but the others use "nxLog_App.h".
So I change it to actual file name with Middleware\NXP\hostLib\libCommon\log\nxLog_App.h.